net-misc/urlscan/urlscan-0.9.1.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 2019 Gentoo Authors
```

Comment:
Why don't you use the default header? Commit with repoman to fix this automatically.

Reply:
It was generated from the ebuild emacs mode, and I always repoman + repoman commit so it's strange that it doesn't picked up. Fixed.

Reply:
Hmm, then I guess it wasn't technically wrong. Just something I haven't seen before, sorry :)